### PR TITLE
fix(php-laravel): fix host, jwt secret key, and command order

### PR DESCRIPTION
**Pull request by sunix**
### What does this PR do?
Fix the php lavarel devfile
- listen to 0.0.0.0 instead of 127.0.0.1 so could directly access to the app with the exposed route
- clarify commands order to be executed
- fix jwt secret generation command

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18441

### How to test this PR?
Could be tested with https://sunix-che-devfile-registry-fix-php-gyd67.surge.sh/devfiles/php-laravel/devfile.yaml
or setting the plugin registry to https://sunix-che-devfile-registry-fix-php-gyd67.surge.sh

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: devfiles/php-laravel/devfile.yaml [hunk 1]
```diff
@@ -49,42 +49,42 @@ components:
   mountSources: true
 commands:
 -
-  name: Start Artisan Web Server
+  name: 6. Start Artisan Web Server
```
Comment (benoitf) [inline devfiles/php-laravel/devfile.yaml:52]: if it's the step 6, maybe we move the command definition at the bottom ?

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
probably need to backport it to 7.28.x

**Comment by sunix** [comment]
@ericwill ok to backport ?

**Comment by ericwill** [comment]
> @ericwill ok to backport ?

Yep!

**Pull request merged**